projects/systelab-components/src/lib/datepicker/datepicker.component.spec.ts

```diff
@@ -183,6 +185,23 @@ describe('Systelab DatepickerComponent', () => {
 		expect(isRedBackground(fixture))
 			.toBeFalsy();
 	});
+
+	it('should show date format on error by default', () => {
+		enterText(fixture, '20/02/1986');
+		expect(isPlaceholderEmpty(fixture))
```

Comment:
Instead to check if placeholder is not empty would be better get the value and be sure if it is what you expect (error message + pattern)